src/components/campaigns/Summary/Stat/style.scss

```diff
@@ -0,0 +1,6 @@
+.CampaignSummaryStat--loading {
```

Comment:
I do want the height to avoid things shifting around when loading or loading is complete